src/Features/Blockcore.Features.Wallet/Api/Controllers/WalletRPCController.cs

```diff
@@ -178,26 +178,28 @@ public NewAddressModel GetNewAddress(string account = "", string addressType = "
 
             if (!string.IsNullOrEmpty(addressType))
             {
-                // Currently segwit and bech32 addresses are not supported.
-                if (!addressType.Equals("legacy", StringComparison.InvariantCultureIgnoreCase))
-                    throw new RPCServerException(RPCErrorCode.RPC_METHOD_NOT_FOUND, "Only address type 'legacy' is currently supported.");
+                // Currently segwit addresses are not supported.
```

Comment:
Doesn't Bech32 imply segwit?

Reply:
Maybe should clarify as p2sh-segwit addresses are not supported. Not sure we need to worry about them as bech32 seems to have been well adopted